### PR TITLE
OSDOCS#6390: Delete deployments running under the cert-manager namespace

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OSDOCS-6390

Link to docs preview: https://60837--docspreview.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/cert-manager-operator-uninstall.html#cert-manager-remove-resources-console_cert-manager-operator-uninstall

QE review:
- [x] QE has approved this change.

SME review:
- [x] SME has approved this change.

Additional information:

### CODE REVIEW

**Comment by swghosh** [comment]
@xenolinux Need to change the sequence a bit from functionality point,
- deployment removal should be first step and please make it non-optional i.e. via avoiding wordings like "alternatively" and or "optionally"
- plus, the section about removal the first part says "optionally", can we change it a bit? Deleting CRDs like certificates, issuers, etc. can be optional but the deployment deletion and that section in general should be mandated

Let me know if you have any further queries.

**Comment by xenolinux** [comment]
> @xenolinux Need to change the sequence a bit from functionality point,
> 
>     * deployment removal should be first step and please make it non-optional i.e. via avoiding wordings like "alternatively" and or "optionally"
> 
>     * plus, the section about removal the first part says "optionally", can we change it a bit? Deleting CRDs like certificates, issuers, etc. can be optional but the deployment deletion and that section in general should be mandated
> 
> 
> Let me know if you have any further queries.

@swghosh Addressed changes. I adjusted the usage of "optional" and the sequence.

**Review by swghosh — APPROVED** [review]
Minor nit, otherwise lgtm.
Thanks again!

**Comment by swghosh** [comment]
/lgtm
@xingxingxia please review for QE approval, TIA!

File: modules/cert-manager-remove-resources-console.adoc [hunk 1]
```diff
@@ -17,7 +17,26 @@ Optionally, after uninstalling the {cert-manager-operator}, you can remove its r
...
+
+.. Navigate to *Workloads* -> *Deployments*.
+
+.. Click the deployment that you want to delete.
+
+.. Click the *Actions* drop-down menu, and select *Delete Deployment* to see a confirmation dialog box.
+
+.. Click *Delete* to delete the deployment.
+
+. Delete deployments of the cert-manager components such as `cert-manager`, `cainjector` and `webhook` present in the `cert-manager` namespace by using the command-line interface (CLI).
```
Comment (xingxingxia) [inline modules/cert-manager-remove-resources-console.adoc:32]: If previous `Remove the deployments of the cert-manager components ...` in web console is done, then it already does what `Delete deployments of the cert-manager components ...` by CLI will do. By console and by CLI are equivalent.
```suggestion
. Alternatively, delete deployments of the cert-manager components such as `cert-manager`, `cainjector` and `webhook` present in the `cert-manager` namespace by using the command-line interface (CLI).
```

**Review by xingxingxia — CHANGES_REQUESTED** [review]
LGTM from QE side except below comments.

**Comment by xenolinux** [comment]
@xingxingxia Addressed comments

**Comment by xingxingxia** [comment]
LGTM from QE side.
/lgtm

File: modules/cert-manager-remove-resources-console.adoc [hunk 2]
```diff
@@ -17,7 +17,26 @@ Optionally, after uninstalling the {cert-manager-operator}, you can remove its r
...
+.. Click *Delete* to delete the deployment.
+
+. Alternatively, delete deployments of the cert-manager components such as `cert-manager`, `cainjector` and `webhook` present in the `cert-manager` namespace by using the command-line interface (CLI).
++
+[source,terminal]
+----
+$ oc delete deployment -n cert-manager -l app.kubernetes.io/instance=cert-manager
+----
+
+. Optional: Remove CRDs that were installed by the {cert-manager-operator}:
```
Comment (snarayan-redhat) [inline modules/cert-manager-remove-resources-console.adoc:39]:
```suggestion
. Optional: Remove the custom resource definitions (CRDs) that were installed by the {cert-manager-operator}:
```

**Review by snarayan-redhat — COMMENTED** [review]
Looks good. Few suggestions.

**Pull request merged**